sdk/messaging/azeventhubs/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Release History
 
-## 0.2.1 (2022-11-08)
+## 0.2.1 (2022-11-10)
```

Comment:
This should bump to 0.3.0 for the breaking change.

Reply:
Done!